File: Array_de_dos_dimenciones.js
const array = 
[
    //Array papuuuu
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
];
//Optenemos el tamaño y las filas papu
for (let fila = 0; fila < array.length; fila++)
 {
    let textoFila = "";
    //Otro for para las columnas papu
    for (let columnas = 0; columnas < array.length; columnas++)
    {
        //Aqui se acomulan las variables del texto con un tabulador \t
        textoFila += array[columnas][fila]  + "\t";
    }
    //Aqui se imprime el texto de la fila papu
    console.log(textoFila);
 }

File: Array_de_dos_dimenciones.cs
using System;

namespace Array_de_dos_dimenciones
{
    class program
     {           
        static void Main()
        {
            //ARRAY
                int[,] Array =
                    {
                    {1, 2, 3},
                    {4, 5, 6},
                    {7, 8, 9}
                    };
                    // El GetLenght sirve para obtener el numero de filas UwU
                    for(int fila =0; fila < Array.GetLength(0); fila++)
                    {
                        // Aqui ahora el GtLenght obtiene las columnas UwU
                       for(int columna = 0; columna < Array.GetLength(1); columna++)

                          {
                            //Aqui se imprime la matiz de dos dimensiones (El \t sirve para que se vea bien UwU)
                             Console.Write(Array[columna, fila] + "\t");
                          }
                        //Aqui salta la linea cuando termine de escribirla
                        Console.WriteLine();
                    }

        }
     }
}